### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by yugbit2021** [yugbit2021/C-plus-plus#3]
Adding proper documentations

**Comment by KaKashi1210** [comment]
I can help you with documentation . Explain me more how you would like to do it and assign me  . I will try to help you !

**Comment by yugbit2021** [comment]
Yeah ! Why not  .
I'm also freshie let's collaborate

**Comment by KaKashi1210** [comment]
Great tell me how would you like to document like creating a read me file about steps wise module ?

**Comment by yugbit2021** [comment]
Yeah creating a reading files and adding new and good questions of given
topics

**Comment by KaKashi1210** [comment]
ok got it .
